script/gen_requirements_all.py

```diff
@@ -98,7 +98,8 @@
     'yahoo-finance',
     'pythonwhois',
     'wakeonlan',
-    'vultr'
+    'vultr',
+    'PyRMVtransport'
```

Comment:
Please keep in order

Reply:
I get your point, but there is no obvious order, is there?